WireAPI/Tests/WireAPITests/APIs/ConnectionsAPI/__Snapshots__/ConnectionsAPITests/testGetConnectionsRequest.request-0-v0.txt

```diff
@@ -1,4 +1,5 @@
-- HTTPRequest
-  - path: "/list-connections/"
-  - method: post
-  - body: {"size":500}
+curl \
+	--request POST \
+	--header "Content-Type: application/json" \
+	--data "{\"size\":500}" \
+	"/list-connections/"
```

Comment:
This format is a choice right? I guess we don't want to keep changing it as otherwise we loose safety it provides. Are we moving to the `curl` style in general?

Reply:
the curl style is coming from the snapshot library, and I think we should stick with that instead of the custom one we had before.

Reply:
I think one thing that would have been nice process wise would be to have replaced these in a separate PR. That way they would have provided a guarantee during this refactor. 

I see that as a team we quite often update snapshot tests during refactors and in doing so lose some of the purpose of the tests.